pkgs/development/python-modules/imagededup/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, pythonPackages
+, fetchFromGitHub
+}:
+
+with pythonPackages;
+
+buildPythonPackage rec {
+  pname = "imagededup";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "idealo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-B2IuNMTZnzBi6IxrHBoMDsmIcqGQpznd/2f1XKo1Oa4=";
+  };
+
+  propagatedBuildInputs = with pkgs.python3Packages; [
+    cython
+    torch
+    torchvision
+    pillow
+    tqdm
+    scikit-learn
+    pywavelets
+    matplotlib
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://idealo.github.io/imagededup/";
+    description = "Finding duplicate images made easy";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ stunkymonkey ];
+    platforms = platforms.linux;
```

Comment:
Why linux only? Also Is this a cli tool? If so it should not be in python-modules or pythonPackages

Reply:
linux was set, because i only tested it on linux.
I removed it.

It is not a cli tool. It is a library for being imported in python.